### ISSUE TITLE
Use tmpfs for integration tests on CI

### ISSUE THREAD

**Issue by jesseduffield** [jesseduffield/lazygit#2701]
**Is your feature request related to a problem? Please describe.**
Integration tests on CI could run faster.

**Describe the solution you'd like**
We should be able to speed up our integration tests on CI by placing the test repo directory of each test in a tmpfs volume. Apparently github actions have one we can use by default: see [here](https://stackoverflow.com/questions/75726151/is-it-possible-to-use-tmpfs-on-a-github-workflow-service)

**Describe alternatives you've considered**
None

**Comment by gustavopcassol** [comment]
This sounds interesting. May I look into it?

**Comment by stefanhaller** [comment]
Of course, go for it! I don't think we need to ask for permission to work on something... 😄
